### PR TITLE
An attempt to make the Authentication docs clearer

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
There is still a lack of clear definition between  ‘API Key’, ‘Authentication Token' and ‘Access Token’, but this seems to be a deeper discussion that I'm not sure one document can clear up. So again this is a first draft…

### CODE REVIEW

File: docs/references/authentication.html.haml [hunk 1]
```diff
@@ -58,38 +61,38 @@ page: :docsAuthentication
...
+  These two APIs use an API key that you can create [using the Content Management API](/developers/docs/references/content-management-api/#/reference/api-keys/create-an-api-key).
 
-  These two APIs use the access token which you can create [using the Content Management API](/developers/docs/references/content-management-api/#/reference/api-keys/create-an-api-key).
+  You can also create API keys using the [Contentful web app](https://app.contentful.com). Open the space that you want to access (the _top left corner_ lists all spaces), and navigate to the _APIs_ area. Open the _API keys_ section and create your first token.
 
-  Alternatively, you can also create API keys by using the [Contentful web app](https://app.contentful.com). Open the space that you want to access (top left corner lists all the spaces), and navigate to the *APIs* area. Open the *API keys* section and create your first token. Done.
+  ![API Keys in Web App](api-keys-webapp.png)
 
-  ## The management API
+  ## The content management API
```
Comment (ohookins) [inline docs/references/authentication.html.haml]: Shouldn't that be `Content Management API` everywhere?

File: docs/references/authentication.html.haml [hunk 2]
```diff
@@ -116,44 +119,34 @@ page: :docsAuthentication
...
+  If you are creating apps for changing content stored in Contentful, you will need to create a custom OAuth application.
 
-  Having your own OAuth 2.0 application has a number of benefits:
+  An OAuth 2.0 application has a number of benefits:
 
-  - OAuth 2.0 access tokens will be linked to your app
-  - You can request the right OAuth 2.0 scopes for your application (`content_management_read` or `content_management_manage`)
-  - You can specify a custom redirect URI that will receive the access token as part of the URI's hash fragment
-  - You can specify a custom name and description
+  - OAuth 2.0 access tokens are linked to your app.
```
Comment (ohookins) [inline docs/references/authentication.html.haml]: "are issued to your app" ?

Comment (ChrisChinchilla) [inline docs/references/authentication.html.haml]: @ohookins I don't think so, this relates to the earlier comments (i think), in saying that each token is linked to an app (in terms of usage), they are sent as well, but this is more commentary on the management of your apps.

File: docs/references/authentication.html.haml [hunk 3]
```diff
@@ -58,38 +61,38 @@ page: :docsAuthentication
...
+  You can also create API keys using the [Contentful web app](https://app.contentful.com). Open the space that you want to access (the _top left corner_ lists all spaces), and navigate to the _APIs_ area. Open the _API keys_ section and create your first token.
 
-  Alternatively, you can also create API keys by using the [Contentful web app](https://app.contentful.com). Open the space that you want to access (top left corner lists all the spaces), and navigate to the *APIs* area. Open the *API keys* section and create your first token. Done.
+  ![API Keys in Web App](api-keys-webapp.png)
 
-  ## The management API
+  ## The content management API
 
-  To access the management API – that is, to put content in Contentful from your apps – you need to get an OAuth 2.0 token that would represent your account. Naturally, this token will have the same rights as the owner of the account.
+  To access the content management API and store content created in your apps, you need an OAuth 2.0 token that represents your account. This token will have the same rights as the owner of the account.
```
Comment (ChrisChinchilla) [inline docs/references/authentication.html.haml]: @ohookins Agreed and will change.

File: docs/references/authentication.html.haml (hunk 1 above)
Comment (ChrisChinchilla) [inline docs/references/authentication.html.haml]: @ohookins Yup, we'll scare @eugenekudashev-tmp but this is a 'brand name' ;)

**Pull request merged**